packages/components/src/NewDataGrid/Row/Row.tsx

```diff
@@ -63,17 +103,51 @@ export type RowProps<TData extends Record<string, unknown>> = {
 export const Row = <TData extends Record<string, unknown>>(
   props: RowProps<TData>,
 ) => {
-  const { isDisabled, checkboxProps, rowProps, tooltipProps } = useLogic(props);
+  const {
+    isOpen,
+    handleToggle,
+    checkboxProps,
+    rowProps,
+    tooltipProps,
+    nestedChildrenProps,
+  } = useLogic(props);
 
-  const { isSelectable, gridColumns, children } = props;
+  const {
+    row,
+    options,
+    isSelectable,
+    gridColumns,
+    isInitialExpanded,
+    expandedLevel,
+    level,
+    nestedChildren,
+    initialOpenedNestedChildrenCount,
+    columns,
+    emptyCellValue,
+    selectedRows,
+    activeRowId,
+    keyId,
+    onSelectRow,
+    onRowClick,
+  } = props;
+
+  const { isDisabled, isDisabledLastCell = true } = options || {};
+
+  const startAdornmentRender = () => {
```

Comment:
https://kaluga-astral.github.io/style-guide/docs/rules/naming/functions#%D1%84%D1%83%D0%BD%D0%BA%D1%86%D0%B8%D0%B8-%D0%B8-%D0%BC%D0%B5%D1%82%D0%BE%D0%B4%D1%8B-%D0%B4%D0%BE%D0%BB%D0%B6%D0%BD%D1%8B-%D0%BD%D0%B0%D1%87%D0%B8%D0%BD%D0%B0%D1%82%D1%8C%D1%81%D1%8F-%D1%81-%D0%B3%D0%BB%D0%B0%D0%B3%D0%BE%D0%BB%D0%B0